<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SequenceSearchWidget</class>
 <widget class="QWidget" name="SequenceSearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QPushButton" name="changeDatabase">
     <property name="text">
      <string>Change Databse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="scanProgress">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font-size: 26px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Sequence Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QPlainTextEdit" name="sequenceText"/>
     <widget class="QTableView" name="resultsTable"/>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="rescanButton">
     <property name="text">
      <string>Rescan Files</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="showErrorsButton">
     <property name="text">
      <string>Show Errors</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
